Formula/goredo.rb

```diff
@@ -0,0 +1,27 @@
+class Goredo < Formula
+  desc "Go implementation of djb's redo, a Makefile replacement that sucks less"
+  homepage "http://www.goredo.cypherpunks.ru/"
+  url "http://www.goredo.cypherpunks.ru/download/goredo-0.11.0.tar.zst"
```

Comment:
Is there no tar.gz?

Reply:
Correct, only zstd http://www.goredo.cypherpunks.ru/Install.html 

Used by default in Arch and Fedora IIUC